test/test_sparse.py

```diff
@@ -2531,11 +2526,11 @@ def test_sparse_add_coalesce(self, device, dtype):
         self.assertFalse(z._indices().numel() != 2 and z.is_coalesced())
 
     @onlyCUDA
-    def test_storage_not_null(self):
-        x = torch.cuda.sparse.FloatTensor(2)
+    def test_storage_not_null(self, device):
+        x = torch.empty(2, dtype=torch.float32, layout=torch.sparse_coo, device=device)
```

Comment:
Let's use the sparse constructors. I think this is supposed to go away @amjames?

Reply:
Yes, I am working on a change that will deprecate user facing `empty` and `empty_like` for sparse layouts. 

You should use the constructors directly, or `zeros` would be the closest replacement here.

Reply:
On second thought, we can also use `sparse_coo_tensor` constructor:
```python
x = torch.sparse_coo_tensor((2,), dtype=torch.float32)
```
which is the shortest variant among other options.